src/com/mambu/apisdk/util/RequestExecutorImpl.java

```diff
@@ -170,8 +179,13 @@ public ByteArrayOutputStream executeRequest(String urlString, ParamsMap params,
 		// Mambu may handle API requests differently for different Application Keys
 		params = addAppKeyToParams(params);
 
-		
-		HttpClient httpClient = new DefaultHttpClient();
+		System.setProperty("org.apache.commons.logging.simplelog.log.org.apache.http.client.protocol.ResponseProcessCookies", "fatal");
```

Comment:
These two strings are also used in the method above. Can we extract them as constants? Moreover, maybe it would be good to have another method that creates the http client, because it looks like these two lines are duplicated in both methods.

Reply:
Right, just extracted method. Thanks! ;)

Reply:
github